### DISCUSSION TITLE
When I set 'accept' on the dropzone, the file browser doesn't work

### DISCUSSION THREAD

**Discussion by SharonGilmore** [react-dropzone/react-dropzone#1198] [Q&A] [answered]
I've set up a dropzone as follows:
let accept: Accept = {
    'application/pdf': ['.pdf', '.PDF'],
    'video/mp4': ['.mp4', '.MP4'],
  };

  return (
    <>
      <Dropzone onDrop={onDrop} accept={accept}>...</Dropzone><>)

When I drag and drop files onto the Dropzone it works fine - valid files are accepted and invalid files are rejected.  However, when I click to open the file browser, it will only allow me to select PDF files - all other types (including mp4) are greyed out (see attached screenshot)).  If I switch accept to put 'video/mp4' first, then the file browser only lets me pick mp4 files.  How do I set it so that both pdf and mp4 are allowed?

![Uploading Screenshot 2022-05-23 at 13.21.46.png…]()

**Comment by brianlovin** [comment]
I just discovered this same bug on my end — whichever option is listed _first_ in the `accept` property is what the file picker will show as enabled files. Drag-n-drop works just fine though. Video repro:

https://user-images.githubusercontent.com/1923260/170519917-23cf28a1-16db-4843-bc3a-ac3b7046e98e.mp4

**Comment by brianlovin** [comment, marked as answer]
@SharonGilmore I was able to find the answer here: 

https://github.com/react-dropzone/react-dropzone/issues/1191#issuecomment-1121245916

**Reply by SharonGilmore** [reply]
Thank you - that solved it for me too
